src/shared/GlobalTopBanners/TrialBanner/TrialBanner.tsx

```diff
@@ -24,13 +24,18 @@ const determineDateDiff = ({
   return dateDiff
 }
 
+const startsWithPlan = (pathName: string): boolean => {
+  return pathName.startsWith('/plan')
+}
+
 interface UrlParams {
   provider?: string
   owner?: string
 }
 
 const TrialBanner: React.FC = () => {
   const { provider, owner } = useParams<UrlParams>()
+  const pathStartsWithPlan = startsWithPlan(useLocation().pathname)
```

Comment:
One small thing, can you pull out `useLocation` here and assign it directly to a variable?